docs/versioning_integration.md

```diff
@@ -73,6 +77,9 @@ class BlogCMSConfig(CMSAppConfig):
             preview_url=get_preview_url
         ),
     ]
+    versioning_add_to_confirmation_context = {
+        'unpublish': [stories_about_intelligent_cats],
```

Comment:
@chaosk How would you feel about changing this to `'unpublish': {'cats': stories_about_intelligent_cats, 'dogs': dog_stories}`?
This would make the context dict something like `{..., 'extra_context': {'cats': ..., 'dogs': ...}}` and the template:
```
{% for thing in extra_context.values %}
    {{thing}}
{% endfor %}
```
But would mean that a project could override that template and reference vars like `{{extra_context.cats}}` and `{{extra_context.dogs}}`

Or we can leave this as a list as it is now.

Reply:
Am still thinking if this is the correct approach, or should that function return a rendered template?

But yes, changing the structure to a dict makes sense